app/views/report/_report_modal.erb

```diff
@@ -0,0 +1,23 @@
+<div class="modal fade" id="reportModal" tabindex="-1" role="dialog">
```

Comment:
I created an erb file instead of a haml one, thinking about a possible future where everything is translated in templates. ERB is HTML so the migration should be easier, and it's easier for me to write than HAML.